.dockerignore

```diff
@@ -0,0 +1,89 @@
+# Git
+.git
+.gitignore
+.gitattributes
+
+
+# CI
```

Comment:
Avoid boilerplate everything-that-might-exist ignore files like this. Ignore the things you actually have in the project and want to skip.